README.md

```diff
@@ -6,51 +6,43 @@ We run an [internal server](./internal-server.md) so private datasets can be kep
 
 We have have general experience [tracking neuroimaging data with git-annex](./git-annex.md).
 
-The data curation scripts should be placed inside the BIDS datasets, under the `code/` folder. See [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code). For more convenience, you can create a PR with a curation script that you are working on, so that others can give feedback; once the script is validated, you can simply close the PR and delete the branch without merging.
+# ⚠️ Every dataset should have the following files 
 
-Every dataset should have a README inside it. Example:
-~~~
-# <NAME OF DATASET>
-
-<PLEASE MODIFY>This is an MRI dataset acquired in the context of the XX project. It also contains manual segmentation of MS lesions from expert raters.
-
-Dataset shared by: <NAME AND EMAIL>
-<IF THERE WAS EMAIL COMM>Email communication: <DATE OF EMAIL AND SUBJECT>
-<REPOSITORY OF PROJECT/MODEL, etc>Repository: https://github.com/ivadomed/model_seg_ms_mp2rage
+```
+├── README.md
+├── participants.json
+├── participants.tsv
+├── dataset_description.json
+└── code
+    └── <script_you_used_for_dataset_curation>.py
+```
 
-<IF THERE ARE UPDATES IN THE DATASET:>2022-10-30: Added new data from INsIDER_SCT_Segmentations_COR
+For details, see [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code).
 
-## dataset structure
+## `README.md`
 
-participants.tsv: Include the following columns:
-- 'pathology': can take values listed in https://docs.google.com/spreadsheets/d/1yjcA8Z0COn4OZxusIDHjStH2DpeXvscsj-aWE2X-_sg/edit#gid=0
-- 'data_id' is the name center_study_subject used to locate the unprocessed data as under duke/mri/.
-- 'institution_id' is used to identify CenterStudy.
-- others? XXX (Jan pls continue)
+The `README.md` is a [markdown](https://markdown-guide.readthedocs.io/en/latest/index.html) file describing the dataset in more detail.
 
+<details><summary>README.md template:</summary>
 
-XXX Jan contnue -- asju julien to check afterward
+```
+# <NAME OF DATASET>
 
+This is an MRI dataset acquired in the context of the <XYZ> project. 
+It also contains manual segmentation of <MS lesions/tumors/etc> from <one/two/or more> expert raters. 
+Segmentation are located under the derivatives folder.
 
+## contact person
 
-CONVENTION FOR JSON METADATA
-----------------------------
+Dataset shared by: <NAME AND EMAIL>
+<IF THERE WAS EMAIL COMM>Email communication: <DATE OF EMAIL AND SUBJECT>
+<REPOSITORY OF PROJECT/MODEL, etc>Repository: https://github.com/<organization>/<repository_name>
 
-UNDER SUBJECTS:
-{
-"Metadata": 
-  {
-  "added_by": "Name of the person who added this data in the database",
-  "added_on": "YYYY-MM-DD",
-  "contact": "URL to forum/issue AND/OR name AND/OR anything relevant to trace back the image",
-  }
-}
+<IF THERE ARE UPDATES IN THE DATASET:><YYYY>-<MM>-<DD>: Added new data from <centre ABC>
```

Comment:
Is a changelog part of the bids spec? `git` provides better changelogs.

If we do need to duplicate the changelog into the README, we should mandate that it's done *via a script*, the way we do e.g. for [sct](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/master/.github/workflows/automate-changelog-PR.yml) or like [this common tool does it](https://www.npmjs.com/package/generate-changelog).

But I would not put a changelog in the README. Let's encourage people to appreciate git. The work I am doing in https://github.com/neuropoly/computers/issues/167 will help! It will give us a nice accessible frontend to our datasets complete with their history a click away. You can see my demo at https://data.dev.neuropoly.org/neuropoly/spine-generic-single/:

![Screenshot 2022-12-06 at 17-38-14 spine-generic-single](https://user-images.githubusercontent.com/987487/206039269-5e6212b2-7601-493a-a6b2-cfd96fd97f88.png)

![Screenshot 2022-12-06 at 17-40-15 spine-generic-single](https://user-images.githubusercontent.com/987487/206039471-20831fdb-9f31-4b28-b610-6aa60b3446c8.png)

I am planning to replace the command-line-only data.neuro.polymtl.ca with this software, within a couple months.

Reply:
`CHANGES` is part of the bids spec, but as an optional file ([spec link](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#changes)). So I think it would be reasonable to omit the `CHANGES` file and include something like "please see `git log` for a history of changes" in `README.md`.